### ISSUE TITLE
CellExternalCopyManager: Copying cells selected with ctrl+A adds a header row between each row

### ISSUE THREAD

**Issue by Exagone313** [6pac/SlickGrid#669]
Hi,

When using the plugin CellExternalCopyManager, you can copy/paste cells to/from a spreadsheet software.

To select multiple cells, you can either click on a cell and drag the mouse to another cell, or you can use CTRL+A to select all the cells at once. Note that the behavior with dragging the mouse was not possible in previous SlickGrid versions, making CTRL+A the way our users copy the cells (we recently upgraded from an older SlickGrid version to be able to upgrade jQuery).

When using CTRL+A, the CellExternalCopyManager plugin receives (via the CellSelectionModel plugin) a list of ranges for each row, making it add a header row between each row.

I'm not able to find in the source code where it actually creates a list of ranges for each row.

I think the real fix would be to create a single range when selecting with CTRL+A, the alternative would be to prevent CellExternalCopyManager from adding multiple header lines (but if it's possible to create a selection that is not a rectangle, it would break this usage).

**Comment by Exagone313** [comment]
This is odd. It probably means we use a particular plugin to be able to use CTRL+A to select all cells (instead of the page).

We include these JS files in our application:
```
/lib/jquery.event.drag-2.3.0.js
/slick.editors.js
/slick.formatters.js
/slick.grid.js
/plugins/slick.autotooltips.js
/plugins/slick.cellselectionmodel.js
/plugins/slick.cellrangedecorator.js
/plugins/slick.cellrangeselector.js
/plugins/slick.cellcopymanager.js
/plugins/slick.cellexternalcopymanager.js
```

**Comment by 6pac** [comment]
Had a quick look, can't find any sign of the 'a' key being detected in any of the above. I find WinMerge to be an excellent compare tool, perhaps try loading the file structures in and comparing them to the latest repo, or the vanilla repo you started with. That should show you what's changed. Or of course Git, if you use it!

**Comment by ghiscoding** [comment]
I don't think you should or even can use `cellcopymanager` and `cellexternalcopymanager` at the same time, you need to choose which of the copy plugin you want to use.

**Comment by Exagone313** [comment]
I haven't been able to find the underlying code and our final users doesn't need to use CTRL+A to select cells. Closing.
